### PR TITLE
hostess 0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by rogeriopradoj**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

[#30contribs](https://rogeriopradoj.com/2015/06/24/30contribs-cheguei-nos-30-e-quero-de-presente-pull-requests-e-contribuicoes-para-projetos-open-source-e-da-comunidade/)

### CODE REVIEW

File: Formula/hostess.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  desc "Idempotent command-line utility for managing your /etc/hosts file"
+  homepage "https://github.com/cbednarski/hostess"
+  url "https://github.com/cbednarski/hostess/archive/v0.2.1.tar.gz"
+  sha256 "3be0198f358d49aa9b17fd8622ab5d4e3732dc45226fe1804f245a0773504aee"
+  head "https://github.com/cbednarski/hostess.git"
+
+  depends_on "go" => :build
+
+  def install
+    (buildpath + "src/github.com/cbednarski/hostess").install Dir[buildpath/"*"]
```
Comment (DomT4) [inline Formula/hostess.rb]:
```ruby
ENV["GOPATH"] = buildpath
dir = buildpath/"src/github.com/cbednarski/hostess"
dir.install buildpath.children

cd dir do
  system "go", "install"
end
bin.install "bin/hostess"
```

**Review by DomT4 — CHANGES_REQUESTED** [review]

File: Formula/hostess.rb (hunk 1 above)
Comment (rogeriopradoj) [inline Formula/hostess.rb]: Hi, @DomT4 and @zmwangx , `cd dir do` doesn't work. Here's the change that is closer to your request and works fine:

```
    ENV["GOPATH"] = buildpath
    dir = buildpath/"src/github.com/cbednarski/hostess"
    dir.install buildpath.children
    
    cd dir/"cmd/hostess" do
      system "go", "install"
    end
    bin.install "bin/hostess"
```

Should I send a new commit, or amend and force a push to this same branch/pull request?

Comment (rogeriopradoj) [inline Formula/hostess.rb]: New commit: 3731b53.

**Comment by MikeMcQuaid** [comment]
@DomT4 👍 ?

**Review by DomT4 — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by rogeriopradoj** [comment]
@MikeMcQuaid My pleasure!

I'm very involved with community here in São Paulo, Brazil, and I'm a big advocate of contributing with open source projects in the user groups that I join and help to organize as well (as @phpsp and @vagrantsp). Thank you for maintaining Homebrew and make your lives easier!

**Comment by MikeMcQuaid** [comment]
@rogeriopradoj Thanks for the kind words!

**Pull request closed without merging**